x/gov/types/content.go

```diff
@@ -17,6 +17,7 @@ const (
 // information such as the title and description along with the type and routing
 // information for the appropriate handler to process the proposal. Content can
 // have additional fields, which will handled by a proposal's Handler.
+// TODO Try to unify this interface with types/module/simulation
```

Comment:
should TODO's have accompanying issues in the sdk, so we dont forget about them? @alessio @alexanderbez